FRENCH/src/ch08-00-common-collections.md

```diff
@@ -0,0 +1,68 @@
+<!--
+# Common Collections
+-->
+
+# Les collections standard
+
+<!--
+Rust’s standard library includes a number of very useful data structures called
+*collections*. Most other data types represent one specific value, but
+collections can contain multiple values. Unlike the built-in array and tuple
+types, the data these collections point to is stored on the heap, which means
+the amount of data does not need to be known at compile time and can grow or
+shrink as the program runs. Each kind of collection has different capabilities
+and costs, and choosing an appropriate one for your current situation is a
+skill you’ll develop over time. In this chapter, we’ll discuss three
+collections that are used very often in Rust programs:
+-->
+
+La bibliothèque standard de Rust apporte quelques structures de données très
+utiles qui s'appellent les *collections*. La plupart des autres types de données
+représentent une seule valeur précise, mais les collections peuvent avoir
+plusieurs valeurs. Contrairement aux types tableau et tuple qui sont intégrés,
```

Comment:
The turn of phrase is misleading, reading to the end of this line, I could understand that collections are not built-in. I'd suggest dropping the precision:
```suggestion
plusieurs valeurs. Contrairement aux tableaux et aux tuples,
```